### ISSUE TITLE
Is this library dead?

### ISSUE THREAD

**Issue by n33pm** [timolins/react-hot-toast#380]
Over 100 issues are open 29 PRs.... 🥺

**Comment by fellypsantos** [comment]
maybe... I did a fork and adapted a bit of code to my needs, and upload again to npm , that's too bad cuz it's a great lib, would be great if the PR's get approved.

**Comment by memark** [comment]
Yeah, it's too bad. Are there any good alternatives available? (Or forks for that matter.)

**Comment by marcosarce** [comment]
This package had great a great checkmark animation. Here is a popular alternative in active development: https://github.com/emilkowalski/sonner

**Comment by timolins** [comment]
I'm _finally_ planning to give react-hot-toast some much deserved love over the Christmas break.

**Comment by maciejstepie** [comment]
bump
